### DISCUSSION TITLE
Using tolgee with SvelteKit - fetch warning

### DISCUSSION THREAD

**Discussion by planetaska** [tolgee/tolgee-platform#1651] [Q&A]
When using tolgee in a SvelteKit project, I got a warning message like this:

````
[Warning] Loading https://app.tolgee.io/v2/projects/*******/translations/en using `window.fetch`. 
For best results, use the `fetch` that is passed to your `load` function: 
https://kit.svelte.dev/docs/load#making-fetch-requests
````

I guess it has to do with the load function in SvelteKit. Is this being looked into?

**Comment by stepan662** [comment]
Hey, what exactly is your setup? I was testing this with our example app with newest svelte and svelte-kit, but I didn't get the warning.

I think it should be solvable by providing static data to Tolgee which you can load in the `load` function based on user locale decided on the server.

**Reply by planetaska** [reply]
Hi! My setup is just like what the docs suggested:

In my root layout (+layout.svelte):
````
<script lang="ts">
  import { TolgeeProvider, Tolgee, DevTools, FormatSimple } from '@tolgee/svelte';
  import enLang from "../i18n/en.json"
  import jaLang from "../i18n/ja-JP.json"
  
  const tolgee = Tolgee()
      .use(DevTools())
      .use(FormatSimple())
      .init({
        language: 'en',
        availableLanguages: ['en', 'ja-JP'],
        apiUrl: import.meta.env.VITE_TOLGEE_API_URL,
        apiKey: import.meta.env.VITE_TOLGEE_API_KEY,
        staticData: {
          en: enLang,
          'ja-JP': jaLang
        }
      });
</script>

<TolgeeProvider tolgee={tolgee}>
  <div class="w-screen h-screen flex items-center justify-center" slot="fallback">Loading...</div>

  <slot />
</TolgeeProvider>
````

This is a regular SvelteKit project created with `npm create svelte@latest` and the version is `"@sveltejs/kit": "^1.11.0"`.

So what I should do is to move the `Tolgee().init()` part to the load function (`+layout.ts`), then pass the entire tolgee object back through `LayoutData`? But that seems weird to me, to be honest?


Update: I tried moving the `Tolgee().init()` part to `+layout.ts`, but still getting the same warning.
![スクリーンショット 2023-04-05 13 33 52](https://user-images.githubusercontent.com/1701424/229981867-7678bc39-ddac-40e2-a02a-272dbacd652b.png)
